src/io/psse.jl

```diff
@@ -473,49 +513,67 @@ function psse2pm_transformer!(pm_data::Dict, pti_data::Dict, import_all::Bool)
+PSS(R)E Voltage Source Converter specification.
 """
 function psse2pm_dcline!(pm_data::Dict, pti_data::Dict, import_all::Bool)
     pm_data["dcline"] = []
 
     if haskey(pti_data, "TWO-TERMINAL DC")
-        warn(LOGGER, "Two-Terminal DC Lines are not yet fully supported")
         for dcline in pti_data["TWO-TERMINAL DC"]
+            info(LOGGER, "Two-Terminal DC lines are supported via a simple *lossless* dc line model approximated by two generators.")
             sub_data = Dict{String,Any}()
 
-            power_demand = dcline["MDC"] == 1 ? abs(pop!(dcline, "SETVL")) : dcline["MDC"] == 2 ? abs(pop!(dcline, "SETVL") / pop!(dcline, "VSCHD") / 1000) : 0
-
-            sub_data["qminf"], sub_data["qmaxf"] = calc_2term_reactive_power(power_demand, pop!(dcline, "ANMNR"), pop!(dcline, "ANMXR"))
-            sub_data["qmint"], sub_data["qmaxt"] = calc_2term_reactive_power(power_demand, pop!(dcline, "ANMNI"), pop!(dcline, "ANMXI"))
+            # Unit conversions?
+            power_demand = dcline["MDC"] == 1 ? abs(dcline["SETVL"]) : dcline["MDC"] == 2 ? abs(dcline["SETVL"] / pop!(dcline, "VSCHD") / 1000) : 0
 
             sub_data["f_bus"] = dcline["IPR"]
             sub_data["t_bus"] = dcline["IPI"]
-            sub_data["br_status"] = dcline["MDC"] == 0 ? 0 : 1
+            sub_data["br_status"] = pop!(dcline, "MDC") == 0 ? 0 : 1
             sub_data["pf"] = power_demand
             sub_data["pt"] = power_demand
             sub_data["qf"] = 0
             sub_data["qt"] = 0
             sub_data["vf"] = get_bus_value(pop!(dcline, "IPR"), "vm", pm_data)
             sub_data["vt"] = get_bus_value(pop!(dcline, "IPI"), "vm", pm_data)
 
-            sub_data["pminf"] = dcline["MDC"] > 0.0 ? 0.9 * power_demand : 0.0
-            sub_data["pmaxf"] = dcline["MDC"] > 0.0 ? 1.1 * power_demand : 0.0
-            sub_data["pmint"] = dcline["MDC"] < 0.0 ? 0.9 * power_demand : 0.0
-            sub_data["pmaxt"] = pop!(dcline, "MDC") < 0.0 ? 1.1 * power_demand : 0.0
+            sub_data["pminf"] = 0.0
+            sub_data["pmaxf"] = dcline["SETVL"] > 0 ? power_demand : -power_demand
+            sub_data["pmint"] = pop!(dcline, "SETVL") > 0 ? -power_demand : power_demand
+            sub_data["pmaxt"] = 0.0
+
+            # How to properly use firing angles to calculate qmin, qmax?
+            # sub_data["qminf"], sub_data["qmaxf"] = calc_2term_reactive_power(power_demand, pop!(dcline, "ANMNR"), pop!(dcline, "ANMXR"))
+            # sub_data["qmint"], sub_data["qmaxt"] = calc_2term_reactive_power(power_demand, pop!(dcline, "ANMNI"), pop!(dcline, "ANMXI"))
+
+            # Relaxed qmin,qmax
+            sub_data["qminf"] = min(sub_data["pminf"], sub_data["pmaxf"])
+            sub_data["qmaxf"] = max(sub_data["pminf"], sub_data["pmaxf"])
+            sub_data["qmint"] = min(sub_data["pmint"], sub_data["pmaxt"])
+            sub_data["qmaxt"] = max(sub_data["pmint"], sub_data["pmaxt"])
```

Comment:
Typically `pmin` and `pmax` are positive values, while `qmin`/`qmax` are negative and positive respectively, on a similar order of magnitude of `pmax`.  So I would expect `qmin` to be along the lines of `-max(abs(pmin), abs(pmax))` or `qmin = -qmax`

Reply:
Since you say `pmin`/`pmax` are generally positive, do you think I should also change `pmin` in voltage source converters on line 615-616 to make the minimum zero, instead of `-pmax`?

Reply:
VSC looks good to me for now.  We are implementing it as a bi-directional component.

If I recall correctly, in this case we are assuming it is one directional based on the input data?  So on one side the p range will be (0,x) and it will be (-x,0) on the other side.  In either case the q range can be (-x,x) on both sides.